src/sentry/issues/run.py

```diff
@@ -0,0 +1,149 @@
+import logging
```

Comment:
I isolated to a separate pr here to avoid importing other parts of Sentry before we initialize subprocesses